Sources/Operators/CombineLatestMany.swift

```diff
@@ -8,6 +8,7 @@
 
 import Combine
 
+@available(iOS 13.0, *)
```

Comment:
All tests should stay iOS 13 and up. Only the target itself should be iOS 11+ 
I'm actually not sure it's possible, but worth checking